Pixel tl
Pixel br
string data
float32 yolo_confidence
float32 ocr_confidence
